docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Installing Knative Eventing using YAML files"
+linkTitle: "Install Eventing using YAML"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Eventing.
+You can install the Knative Eventing component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Serving component, see [Installing Serving using YAML files](./install-serving-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
+
+## Install the Eventing component
+
+To install the Eventing component:
+
+1. Install the required custom resources:
+
+   ```bash
+   kubectl apply -f {{< artifact repo="eventing" file="eventing-crds.yaml" >}}
+   ```
+
+1. Install the core components of Eventing:
+
+   ```bash
+   kubectl apply -f {{< artifact repo="eventing" file="eventing-core.yaml" >}}
+   ```
+
+
+## Verify the installation
+
+Monitor the Knative components until all of the components show a `STATUS` of `Running`:
+
+```bash
+kubectl get pods --namespace knative-eventing
+```
+
+
+## Optional: Install a default Channel (messaging) layer
+
```

Comment:
I don't think it's correctly worded here to say default, I would assume this means the channel supplied with a default install, i.e. InMemoryChannel even though I know you probably mean "install a channel and set it as the default type". Things like Kafka have additional requirements afaik like a Kafka cluster and installing additional CRDs. Maybe have someone from @knative/eventing-wg-leads check this part.

Reply:
This wording was already on the page and is in the v0.21 docs (step 3 [here](https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-eventing-component)).